data/json/effects.json

```diff
@@ -1481,6 +1489,28 @@
       [ "Your feel stronger as your anemia starts to improve.", "good" ]
     ],
     "max_intensity": 3,
+    "base_mods": {
+      "str_mod": [ -1 ],
+      "stamina_min": [ -100 ],
+      "stamina_max": [ -200 ],
+      "stamina_chance": [ 900 ],
+      "fatigue_min": [ 10 ],
+      "fatigue_max": [ 20 ],
+      "fatigue_chance": [ 900 ],
+      "h_mod_min": [ -1 ],
+      "h_mod_min_val": [ 0 ],
+      "h_mod_chance": [ 900 ]
+    },
+    "scaling_mods": {
+      "str_mod": [ -2 ],
+      "stamina_max": [ -500 ],
+      "fatigue_max": [ 20 ],
+      "stamina_chance": [ -300 ],
+      "fatigue_chance": [ -300 ],
+      "h_mod_min": [ -1 ],
+      "h_mod_min_val": [ -50 ],
+      "h_mod_chance": [ -200 ]
+    },
```

Comment:
This looks great to me, based on numbers alone. The check would be how long it takes to develop anaemia and how fast these symptoms worsen. Until we have a blood loss system I'd assume it should take a few weeks to months of insufficient dietary iron to progress in anaemia stages.